source/apps/crawl_space/templates/crawl_space/crawl.html

```diff
@@ -153,11 +148,18 @@
         </div>
         {% endif %}
         {% endfor %}
-    {% elif crawl.crawler == 'ache' %}
-        <div class='col-sm-6'>
+
+    {% elif crawl.crawler == "ache" %}
```

Comment:
We need to be *slightly* smarter in the view template.  If the streaming plot services aren't available, we shouldn't tempt the user unnecessarily :)